src/amo/components/OverallRating.js

```diff
@@ -70,12 +104,70 @@ export class OverallRatingBase extends React.Component {
   }
 }
 
-export const mapStateToProps = (state) => ({
-  apiState: state.api,
-  userId: state.auth && state.auth.userId,
-});
+export const mapStateToProps = (state, componentProps) => {
+  const userId = state.auth && state.auth.userId;
+  let userReview;
+
+  // Look for an already saved review by this user for this add-on/version.
+  if (userId && state.reviews) {
+    const allUserReviews = state.reviews[userId];
+    log.info(`Checking state for review by user ${userId},
```

Comment:
Ah, I guess we really want [`oneLine`](https://github.com/declandewet/common-tags#oneline). Doesn't look like we have a babel plugin for it though. Could import it I guess since this isn't in a translation.